pxemgr/ipxe_handlers.go

```diff
@@ -30,18 +30,46 @@ const (
 	qemuKernelFile   = "coreos_production_qemu.vmlinuz"
 
 	defaultProfileName = "default"
+
+	kvmStaticSerial = "0123456789"
```

Comment:
this is hack for kvm running on gridscale/KVM as the serial is static and has this value, if we encounter it we use UUID for server serial instead

Reply:
would be good to put such comments in the code itself :)